variables.md

```diff
@@ -113,6 +118,62 @@ PERF_KERNEL | boolean | false | Enables kernel performance testing.
 PERF_INSTALL | boolean | false | Enables kernel performance testing installation part.
 PERF_SETUP | boolean | false | Enables kernel performance testing deployment part.
 PERF_RUNCASE | boolean | false | Enables kernel performance testing run case part.
+_SECRET_PUBLIC_CLOUD_REST_URL | string | "https://publiccloud.qa.suse.de/vault" | Vault server URL
+_SECRET_PUBLIC_CLOUD_REST_USER | string | "" | Vault server user name
+_SECRET_PUBLIC_CLOUD_REST_PW | string | "" | Vault server user password
+_SECRET_PUBLIC_CLOUD_REST_SSL_INSECURE | boolean | true | Do not validate Vault server certificate
+PUBLIC_CLOUD | boolean | false | All Public Cloud tests have this variable set to true. Contact: qa-c@suse.de
+PUBLIC_CLOUD_ACCOUNT | string | "" | 
```

Comment:
```suggestion
PUBLIC_CLOUD_ACCOUNT | string | "" |  For GCE will set account via `gcloud config set account ' . $self->account`
```

Reply:
In general, maybe we should rename variables which are provider specific. 
We didn't in the past, so we might could reuse them for others. But I would actually like it more, that variables which are not supported by all provides, have a prefix like `PUBLIC_CLOUD_GCE..`. WDYT? (not for this PR!)

Reply:
make sense , but for sure not in this PR :)

Reply:
https://progress.opensuse.org/issues/98379